### PR TITLE
Feat: Add review criteria to Review Form

### PR DESCRIPTION
**Pull request by alejandroclose**
Hi @MichaelKohler ! Please check out this approach to fix issue #220 Its a modal triggered by the "Review Criteria" button in the review section.
Thank you!

Alejandro.

### CODE REVIEW

File: doc/review-sentences.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Review Criteria
+
+### Make sure the sentence meets the following criteria:
+
+1. The sentence must be spelled correctly.
+1. The sentence must be grammatically correct.
+1. The sentence must be speakable.
+1. If the sentence meets the criteria, click the "yes" button on the right.
+1. If the sentence does not meet the above criteria, click the "no" button on the right. If you are unsure about the sentence, you may also skip it and move on to the next one.
```
Comment (MichaelKohler) [inline doc/review-sentences.md]: that doesn't really match the icons and swipe interface anymore, but that's the same in the normal how to, so let's change that outside of this PR.

File: web/css/modal.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  bottom: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, .4);
+  overflow: scroll;
+}
+
+.close-modal {
+  position: absolute;
+  right: 0;
+  top:0;
```
Comment (MichaelKohler) [inline web/css/modal.css]: minor: missing space

File: web/css/modal.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  border-color: transparent;
+  color: black;
+  background-color: var(--main-bg-color);
+}
+
+.no-show {
+  display: none;
+}
+
+.info-button{
```
Comment (MichaelKohler) [inline web/css/modal.css]: minor: missing space

File: web/src/components/modal.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import '../../css/modal.css'
+
+class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ({
+      modalIsOpen: false
+    })
```
Comment (MichaelKohler) [inline web/src/components/modal.jsx]: minor: missing semicolon

File: web/src/components/modal.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  handleModalIsOpen() {
+    this.setState({
+      modalIsOpen: !this.state.modalIsOpen,
+    }
+    )
```
Comment (MichaelKohler) [inline web/src/components/modal.jsx]: minor: can you move this to the line above and add a semicolon?

File: web/src/components/modal.jsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  }
+  render() {
+    const handleModal = this.handleModalIsOpen.bind(this);
+    return (
+      <div>
+        <button className="info-button" onClick={handleModal}>{this.props.text}</button>
+        <div className={this.state.modalIsOpen ? "overlay" : "no-show"} onClick={handleModal}>
+          <div className={this.state.modalIsOpen ? "modal-box" : "no-show"}>
```
Comment (MichaelKohler) [inline web/src/components/modal.jsx]: I think that could just be `className="modal-box"` as the parent node gets hidden if it's not open

File: web/src/components/pages/review.jsx [hunk 7]
```diff
@@ -177,6 +179,10 @@ class Review extends React.Component {
       <div>
         <section>
           <h1>Review Sentences</h1>
+          {/* review criteria modal */}
```
Comment (MichaelKohler) [inline web/src/components/pages/review.jsx]: I'd say this can be removed, as the text right below describes it quite well :)

File: web/src/components/modal.jsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+            <button className="close-modal" onClick={handleModal}>
```
Comment (MichaelKohler) [inline web/src/components/modal.jsx]: this `onClick` is not necessary, as all the clicks on the overlay will handle it

File: doc/review-sentences.md (hunk 1 above)
Comment (alejandroclose) [reply doc/review-sentences.md]: Great! Thanks :)

File: web/src/components/modal.jsx (hunk 6 above)
Comment (alejandroclose) [reply web/src/components/modal.jsx]: Of course!!! Thank you!!! 💃

File: web/src/components/modal.jsx (hunk 8 above)
Comment (alejandroclose) [reply web/src/components/modal.jsx]: 👍

File: package.json [hunk 9]
```diff
@@ -62,6 +62,7 @@
     "ava": "^2.4.0",
     "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
+    "babel-preset-env": "^1.7.0",
```
Comment (MichaelKohler) [inline package.json]: why is this needed?
